tests/integration/agents/test_agent_metrics_integration.py

```diff
@@ -0,0 +1,502 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
```

Comment:
yet another ridiculous file. are you sure this is an integration test with a 1000 mocks? did you review this?